app/models/attached_to_interactive.rb

```diff
@@ -0,0 +1,53 @@
+module AttachedToInteractive
+  def self.included(clazz)
+    clazz.class_eval do
+      # That requires model that includes this module to have interactive_id (integer) and interactive_type (string) attributes.
+      belongs_to :interactive, :polymorphic => true
+
+      attr_accessible :interactive_select_value
+      attr_writer :interactive_select_value
```

Comment:
It seems a little strange that this module is adding the `belongs_to` for the interactive. But it isn't adding the `attr_accessible` for interactive_type and interactive_id